integrations/standalone/src/mock/meta.ts

```diff
@@ -1,45 +1,70 @@
-import type { ProjectVarNode } from '@axonivy/variable-editor-protocol';
+import type { KnownVariables, MetaData } from '@axonivy/variable-editor-protocol';
 
-export const knownVariables: ProjectVarNode = {
-  key: '',
+export const knownVariables: KnownVariables = {
+  namespace: '',
   name: '',
   value: '',
-  type: 'folder',
+  metaData: { type: 'folder' },
   description: '',
   children: [
     {
-      key: 'Amazon',
+      namespace: '',
       name: 'Amazon',
       value: '',
-      type: 'folder',
+      metaData: { type: 'folder' },
       description: '',
       children: [
         {
-          key: 'Amazon.Comprehend',
+          namespace: 'Amazon',
           name: 'Comprehend',
           value: '',
-          type: 'folder',
+          metaData: { type: 'folder' },
           description: 'Amazon comprehend connector settings',
           children: [
             {
-              key: 'Amazon.Comprehend.SecretKey',
+              namespace: 'Amazon.Comprehend',
               name: 'SecretKey',
               value: '<YOUR_SECRET_KEY>',
-              type: 'password',
+              metaData: { type: 'password' },
               description: 'Secret key to access amazon comprehend',
               children: []
             },
             {
-              key: 'Amazon.Comprehend.AccessKey',
+              namespace: 'Amazon.Comprehend',
               name: 'AccessKey',
               value: '<YOUR_ACCESS_KEY>',
-              type: 'string',
+              metaData: { type: 'string' },
               description: 'Access key to access amazon comprehend',
               children: []
             }
           ]
         }
       ]
+    },
+    {
+      namespace: '',
+      name: 'Meta',
+      value: '',
+      metaData: { type: 'folder' },
+      description: '',
+      children: [
+        {
+          namespace: 'Meta',
+          name: 'Enum',
+          value: 'two',
+          metaData: { type: 'enum', values: ['one', 'two', 'three'] } as MetaData,
```

Comment:
In the core, we have a base class `MetaData` just containing the field `type` for meta data types without additional info (`password`, `daytime`) and specific sub classes for meta data types with additional info. E.g. `EnumMetaData` extending `MetaData` and having the additional field `values`.

I assume because the return type of the endpoint only contains the super class `MetaData` but not its sub classes, these classes are not being generated for the protocol type.

I will look into an improvement.